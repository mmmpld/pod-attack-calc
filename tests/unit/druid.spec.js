import breakpoints from './breakpoints.js'
const characterId = 3;

describe('Druid default values', () => {
    it('standard attack', async () => {
        await breakpoints('15', '1.66', { charactersSelected: characterId });
    })
})