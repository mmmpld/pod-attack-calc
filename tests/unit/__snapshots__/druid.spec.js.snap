// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Druid default values standard attack 1`] = `
<tbody>
  <tr class="highlight-current">
    <td>0</td>
    <td>15</td>
    <td>1.66</td>
  </tr>
  <tr class="">
    <td>9</td>
    <td>14</td>
    <td>1.78</td>
  </tr>
  <tr class="">
    <td>18</td>
    <td>13</td>
    <td>1.92</td>
  </tr>
  <tr class="">
    <td>30</td>
    <td>12</td>
    <td>2.08</td>
  </tr>
  <tr class="">
    <td>48</td>
    <td>11</td>
    <td>2.27</td>
  </tr>
  <tr class="">
    <td>75</td>
    <td>10</td>
    <td>2.5</td>
  </tr>
  <tr class="">
    <td>125</td>
    <td>9</td>
    <td>2.77</td>
  </tr>
</tbody>
`;
